assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -790,6 +793,31 @@ const baseSelectors = {
 		const requirementsStatus = state.checkRequirementsResults[ slug ];
 		return requirementsStatus === true ? null : requirementsStatus;
 	},
+
+	/**
+	 * Gets the module's screenWidgetContext.
+	 *
+	 * Returns `null` if there is no registered context string for the given module.
+	 * Returns `string` the registered context string, screenWidgetContext for the given module.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state      Data store's state.
+	 * @param {string} moduleSlug Module slug.
```

Comment:
This isn't accurate as the current `moduleSlug` variable here is an object.

But I think this should remain and the module slug ONLY should be supplied to this function, not the object. The `name` attribute isn't used anywhere and makes this a bit more confusing to read.

Let's update this to take only the actual `slug` string value as an argument.